### ISSUE TITLE
[Feature request] Simple SSL diagnostics for Postgresql, Oracle, MySQL etc. SSL DB services.

### ISSUE THREAD

**Issue by mkgrgis** [dbeaver/dbeaver#5407]
By 
https://github.com/dbeaver/dbeaver/issues/5302

Warnings before connection if there is no conditions
1. Valid format DER or PEM for key
2. Valid format for crt
3. Valid user certificate by dates
4. Valid root certificate by dates
4. The key have no permissions to group and other
5. Equal modulus in user's certificate and user's key
etc.

**Comment by serge-rider** [comment]
Good point. However this is not that easy and require extra user interface.

**Comment by mkgrgis** [comment]
Extra user interface can be a passive checkboxes () near filenames: file exist, file format is ok, file date(crt)/permissions(pem) is ok.
